lib/tapioca/runtime/trackers/mixin.rb

```diff
@@ -81,5 +79,21 @@ def extend_object(obj)
       ) if Module === obj
       super
     end
+
+    private
+
+    # Including or prepending on a singleton class is functionally equivalent to extending the
+    # attached class. Registering the mixin as an extend on the attached class ensures that
+    # this mixin can be found whether searching for an include/prepend on the singleton class
+    # or an extend on the attached class.
+    def register_extend_on_attached_class(constant)
```

Comment:
This functionality was moved from `Listeners::Mixins` so that an include/prepend on a singleton class can also be found as an extend on the attached class.